### PR TITLE
Studio comments off

### PR DESCRIPTION
**Pull request by picklesrus**
Adds in comment status component to show when studio comments are off. 
image

### CODE REVIEW

File: src/views/studio/studio-comments.jsx [hunk 1]
```diff
@@ -74,54 +75,67 @@ const StudioComments = ({
                 <h2><FormattedMessage id="studio.commentsHeader" /></h2>
                 {canEditCommentsAllowed && <StudioCommentsAllowed />}
             </div>
```
Comment (picklesrus) [inline src/views/studio/studio-comments.jsx:77]: This part of the diff is much easier to read if you turn whitespace changes off.

File: src/redux/session.js [hunk 2]
```diff
@@ -142,6 +142,8 @@ module.exports.selectIsSocial = state => get(state, ['session', 'session', 'perm
 module.exports.selectIsEducator = state => get(state, ['session', 'session', 'permissions', 'educator'], false);
 module.exports.selectProjectCommentsGloballyEnabled = state =>
     get(state, ['session', 'session', 'flags', 'project_comments_enabled'], false);
+module.exports.selectStudioCommentsGloballyEnabled = state =>
```
Comment (paulkaplan) [inline src/redux/session.js:145]: @picklesrus I think that this should probably be true by default right until we hear from the session? I've been trying to make sure the path to trying to load stuff is as parallel as possible, and this change would prevent loading the comments list until the user session has been fetched, which isn't necessary.

File: src/views/studio/studio-comments.jsx [hunk 3]
```diff
@@ -74,54 +75,67 @@ const StudioComments = ({
...
-            <div>
-                {shouldShowCommentComposer ?
-                    (commentsAllowed ?
-                        <ComposeComment
+            {!hasFetchedSession || studioCommentsGloballyEnabled ?
```
Comment (paulkaplan) [inline src/views/studio/studio-comments.jsx:78]: Why is the `!hasFetchedSession` here blocking the comments from rendering? possibly related to my other comment about not blocking comment loading with session request

**Review by paulkaplan — CHANGES_REQUESTED** [review]
A couple of questions/requests for changes. I'm happy to talk about the questions IRL.

**Comment by picklesrus** [comment]
This isn't finished (e.g.tests would need to be cleaned up), but I'm curious if you like this approach better

**Comment by paulkaplan** [comment]
I approve of the direction. I like the unit test on studio-comments and probably would want a unit test on the permission selectors as well.

**Comment by picklesrus** [comment]
Simplified the tests in studio-comments and moved them to studio-permissions. Glad they were there because they found a bug in the refactor. Fixed that and resolved some merge conflicts. Ready for another review!

**Review by paulkaplan — APPROVED** [review]
LGTM

**Comment by paulkaplan** [comment]
Sorry about the continuing merge conflicts, there is a lot going on with studio comments right now. Should be settled now for at least a few days, but definitely recommend merging asap

**Comment by picklesrus** [comment]
I added some test cases to handle the single comment mode. If they pass on Circle, I'm merging. :)

**Pull request merged**